### DISCUSSION TITLE
关于全局规则匹配应用的问题

### DISCUSSION THREAD

**Discussion by AIsouler** [gkd-kit/gkd#615] [General]
全局规则默认禁用系统应用，但是有部分系统应用是有开屏广告的，想要在默认禁用系统应用的情况下使全局规则匹配某个系统应用，目前的解决办法是设置```apps: [{ id, enable: true }]```,这样设置后id里的全局规则开关是打开的，但实际上不能生效，需要手动关闭再打开才能生效，请问是不是和全局规则里的```matchSystemApp: false,```冲突了？
日志：https://github.com/gkd-kit/inspect/files/15269881/file.zip

https://github.com/gkd-kit/gkd/assets/57941037/418937e0-221a-452f-91d1-50d8227edee8

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/commit/51e8362c6ea49e9d7ebd6b423a6671fe2f7cafe4
